_posts/2016-01-22-let-vs-instance.markdown

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: "Let vs Instance Variables"
+date: 2016-01-22 10:36:00
+categories: ["rails", "rspec"]
+author: "schmierkov"
+---
+
+Maybe in the past you stumbled over the two different approaches to setup your test variables. One way is the more programmatical approach by using instance variables, usually initialized in a `before` block.
+
+```ruby
+  before do
+    @user = User.create username: 'foo'
+  end
+```
+
+The other option is `let`.
+
+```ruby
+  let(:user) { User.create username: 'foo' }
+```
+
+If you have seen this and you are still not sure, when to use instance variables and when `let`, then you will in the following examples.
+
+For this example I have choosen a simple setup in which I want to create 2 users and check their attributes.
+
+To reset the Database properly after each Rspec configuration for this example looks like this:
```

Comment:
@schmierkov could you try rephrasing this? It's a bit confusing, maybe something like `To properly reset the database after each test, you can use this snippet below`, or something similar.